File: projects/askfin_web/src/types/DifficultyLevel.ts
export const DIFFICULTY_ORDER = [
  "easy",
  "medium", 
  "hard",
] as const;

export type DifficultyLevel = typeof DIFFICULTY_ORDER[number];

export function isHarder(a: DifficultyLevel, b: DifficultyLevel): boolean {
  return DIFFICULTY_ORDER.indexOf(a) > DIFFICULTY_ORDER.indexOf(b);
}

File: projects/GrowMeldAI/Models/DifficultyLevel.swift
enum DifficultyLevel: String, Sendable {  // ✅ Explicit Sendable
    case easy
    case medium
    case hard
}

// Enum ExamMode declared in Models/LocalUser.swift

enum PassStatus: String, Sendable {
    case passed
    case failed
}

// Now AnalyticsEvent is safely Sendable